src/Core/IToolWindowService.cs

```diff
@@ -27,6 +27,7 @@ public interface IToolWindowService
         /// <summary>
         /// Opens specified tool window
         /// </summary>
-        void Show(Guid toolWindowId);
+        /// <param name="activateWindow">True if the window should receive focus, otherwise false</param>
```

Comment:
Based on the doc comment, maybe `focus` is more appropriate?

Reply:
`Activate` is the terminology used in the VS interfaces and documentation so I'd rather not introduce a new term.
I'll change the comment.

Reply:
Previously, calling the method would create the window if it didn`t already exist, bring it to the front and give it the focus.
The new parameter controls whether the window is brought to the front _and_ given the focus.

Reply:
Ok, maybe we should separate it to 2 methods then:

`void CreateAndActivate(Guid toolWindowId)` - which is what we would call everywhere (essentially renaming the Show)
`void Create(Guid toolWindowId)` - which is what we can call in the new taint context.

This way it would be clearer what it does and we won't need to pass `true` param in most cases.

Reply:
I`ll add another method.